roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+- name: Assert st2chatops_hubot_adapter is specified
+  fail:
+    msg: >
+      '"st2chatops_hubot_adapter" must be supplied in roles/st2chatops/defaults/main.yml for st2chatops installation.'
```

Comment:
This error message looks not exactly what we check, according to:
```
when: st2chatops_hubot_adapter not in supported_hubot_adapters
```
conditional.

As I understand we really check if `{{ st2chatops_hubot_adapter }}` is supported, but throw an error saying that `{{ st2chatops_hubot_adapter }}` is not provided in `roles/st2chatops/defaults/main.yml`.